test/e2e/bootstrap_test.go

```diff
@@ -0,0 +1,33 @@
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/openshift/cluster-image-registry-operator/test/framework"
+)
+
+func TestBootstrapFailToUpdateSpec(t *testing.T) {
+	te := framework.Setup(t)
```

Comment:
Use SetupAvailableImageRegistry instead of this bunch of Deploy/Wait/Ensure calls.